tpm2/tpm2_test.go

```diff
@@ -360,35 +360,6 @@ func TestCertify(t *testing.T) {
 	if err := rsa.VerifyPKCS1v15(signerPub.(*rsa.PublicKey), crypto.SHA256, attestHash[:], sig); err != nil {
 		t.Errorf("Signature verification failed: %v", err)
 	}
-
```

Comment:
Similarly here, why is this code being removed?

Reply:
Incorrect merge. Will fix.